Sources/MapboxCoreNavigation/NavigationEventsManager.swift

```diff
@@ -62,8 +76,12 @@ open class NavigationEventsManager {
         return token
     }()
     
-    public required init(dataSource source: EventsManagerDataSource?, accessToken possibleToken: String? = nil, mobileEventsManager: MMEEventsManager = .shared()) {
-        dataSource = source
+    public required init(activeNavigationDataSource: EventsManagerDataSource? = nil,
+                         passiveNavigationDataSource: PassiveNavigationEventsManagerDataSource? = nil,
+                         accessToken possibleToken: String? = nil,
+                         mobileEventsManager: MMEEventsManager = .shared()) {
+        dataSource = activeNavigationDataSource
```

Comment:
Maybe it's better to rename the old `dataSource` to `activeNavigationDataSource`?

Reply:
Thanks I will do it since it's not a public property